advanced_source/static_quantization_tutorial.rst

```diff
@@ -0,0 +1,641 @@
+`here <https://github.com/pytorch/examples/blob/master/imagenet/main.py>`_. 
+
+.. code:: python
+
+    class AverageMeter(object): 
+        """Computes and stores the average and current value""" 
+        def __init__(self, name, fmt=':f'): 
+            self.name = name  
+            self.fmt = fmt  
+            self.reset()  
+
+        def reset(self):  
+            self.val = 0  
+            self.avg = 0  
+            self.sum = 0  
+            self.count = 0  
+
+        def update(self, val, n=1): 
+            self.val = val  
+            self.sum += val * n 
+            self.count += n 
+            self.avg = self.sum / self.count  
+
+        def __str__(self):  
+            fmtstr = '{name} {val' + self.fmt + '} ({avg' + self.fmt + '})' 
+            return fmtstr.format(**self.__dict__) 
+
+
+    def accuracy(output, target, topk=(1,)):  
+        """Computes the accuracy over the k top predictions for the specified values of k"""  
+        with torch.no_grad(): 
+            maxk = max(topk)  
+            batch_size = target.size(0) 
+
+            _, pred = output.topk(maxk, 1, True, True)  
+            pred = pred.t() 
+            correct = pred.eq(target.view(1, -1).expand_as(pred)) 
+
+            res = []  
+            for k in topk:  
+                correct_k = correct[:k].reshape(-1).float().sum(0, keepdim=True)  
+                res.append(correct_k.mul_(100.0 / batch_size))  
+            return res  
+
+
+    def evaluate(model, criterion, data_loader, neval_batches): 
+        model.eval()  
+        top1 = AverageMeter('Acc@1', ':6.2f') 
+        top5 = AverageMeter('Acc@5', ':6.2f') 
+        cnt = 0 
+        with torch.no_grad(): 
+            for image, target in data_loader: 
+                output = model(image) 
+                loss = criterion(output, target)  
+                cnt += 1  
+                acc1, acc5 = accuracy(output, target, topk=(1, 5))  
+                print('.', end = '')  
+                top1.update(acc1[0], image.size(0)) 
+                top5.update(acc5[0], image.size(0)) 
+                if cnt >= neval_batches:  
+                     return top1, top5  
+
+        return top1, top5 
+
+    def load_model(model_file): 
+        model = MobileNetV2() 
+        state_dict = torch.load(model_file) 
+        model.load_state_dict(state_dict) 
+        model.to('cpu') 
+        return model  
+
+    def print_size_of_model(model): 
+        torch.save(model.state_dict(), "temp.p")  
+        print('Size (MB):', os.path.getsize("temp.p")/1e6)  
+        os.remove('temp.p') 
+
+3. Define dataset and data loaders  
+----------------------------------  
+ 
+As our last major setup step, we define our dataloaders for our training and testing set. 
+ 
+ImageNet Data 
+^^^^^^^^^^^^^ 
+
+Please download imagenet dataset by following the instructions at: TBD. The rest of the tutorial assumes that you have imagenet_1k.zip which contains the sample imagenet dataset.
```

Comment:
Replace" Please download imagenet dataset by following the instructions at: TBD. The rest of the tutorial assumes that you have imagenet_1k.zip which contains the sample imagenet dataset.Unzip imagenet_1k.zip to `data` folder. To run the code in this tutorial using the entire ImageNet dataset, please follow the instructions at `here <http://www.image-net.org/download>`_." 

with

" To run the code in this tutorial using the entire ImageNet dataset, first download imagenet by following the instructions at `here <http://www.image-net.org/download>`_. Unzip the downloaded file into the 'data_path' folder."